src/root/views/InformalUpdateReport/index.tsx

```diff
@@ -126,57 +114,74 @@ function InformalUpdateReport(props: Props) {
             compact={undefined}
           />
         }
+        actions={
+          <Button
+            variant='secondary'
+            name={undefined}
+            icons={<IoShareSocialOutline />}
+          >
+            Share
+          </Button>
+        }
       >
-        <Container heading="SITUATIONAL OVERVIEW">
-          <ViewSection>
-            <div>
-              {situationalData.map((item) => (
-                <>
-                  <div className={styles.overviewContainer}>
-                    <div className={styles.overviewLabel}>
-                      {item.name}
-                    </div>
-                    <div className={styles.overviewValue}>
-                      {item.value}
-                    </div>
+        {projectPending ?
+          <BlockLoading />
+          : (
+            <>
+              <Container heading={strings.informalUpdateSituationalOverviewLabel} >
+                <ViewSection>
+                  <div>
+                    {situationalData.map((item, i) => (
+                      <div key={i}>
+                        <div className={styles.overviewContainer}>
+                          <div className={styles.overviewLabel}>
+                            {item.name}
+                          </div>
+                          <div className={styles.overviewValue}>
+                            {item.value}
+                          </div>
+                        </div>
+                        <div className={styles.overviewDescription}>
+                          {item.description}
+                        </div>
+                      </div>
```

Comment:
use ListView if possible, this is a big chunk and would be more readable if done in a separate component